src/app/content/components/CallToActionPopup/styles.ts

```diff
@@ -0,0 +1,124 @@
+import styled from 'styled-components';
+import { Times } from 'styled-icons/fa-solid/Times/Times';
+import Button from '../../../components/Button';
+import { textRegularStyle } from '../../../components/Typography';
+import { textStyle } from '../../../components/Typography/base';
+import theme from '../../../theme';
+
+// tslint:disable-next-line: variable-name
+export const CTAWrapper = styled.div`
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  z-index: 99;
+  width: 100%;
+
+  @media screen and (max-width: 1200px) {
```

Comment:
there is a helper for mobile styles that you should use instead of this. 1200px is the mobile breakpoint but it is not good to use px in media queries